.github/workflows/frontend-shared.yml

```diff
@@ -0,0 +1,34 @@
+name: frontend-shared package
+on:
+  release:
+    types:
+      - published
+jobs:
+  update-frontend-shared-version:
```

Comment:
This is redundant with CI in some respects, so there may be ways to optimize these jobs.